Library/Homebrew/utils/gzip.rb

```diff
@@ -0,0 +1,73 @@
+# typed: true
+# frozen_string_literal: true
+
+# Apple's gzip also uses zlib so use the same buffer size here.
+# https://github.com/apple-oss-distributions/file_cmds/blob/file_cmds-400/gzip/gzip.c#L147
+GZIP_BUFFER_SIZE = 64 * 1024
+
+module Utils
+  # Helper functions for creating gzip files.
+  #
+  # @api private
+  module Gzip
```

Comment:
We don't seem to be treating this as private API? We deprecated the plain `gzip` Kernel extension function, pointed people to the (fully-qualified) `Utils::Gzip.compress` and have made changes to homebrew-core to use it.

Reply:
Yes, sounds like this should be public.